<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrequencyFileGeneratorDialog</class>
 <widget class="QDialog" name="FrequencyFileGeneratorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frequency File Generator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="inputButton">
           <property name="text">
            <string>Select input</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="outputButton">
           <property name="text">
            <string>Select output</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Morphemizer:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
         <item alignment="Qt::AlignLeft">
          <widget class="QComboBox" name="comboBox">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="createFrequencyFileButton">
       <property name="styleSheet">
        <string notr="true">margin-top: 10px;
margin-bottom: 10px;
padding: 5px;</string>
       </property>
       <property name="text">
        <string>Create Frequency File</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
